db/models/topics-model.js

```diff
@@ -5,3 +5,11 @@ exports.fetchAllTopics = () => {
     return result.rows;
   });
 };
+
+exports.fetchArticleId = (article_id) => {
```

Comment:
I think this is duplicated code that isn't used anywhere, can you please remove 🙂